src/metatrader/account_mt5.py

```diff
@@ -21,11 +22,16 @@ def get_account_profit(self) -> float:
         profit = account_info_dict['profit']
         return profit
     
-    def get_positions(self) -> tuple: # Note on 1.21.24 that MT5 returns positions as a named Tuple
-        positions = mt5.positions_get()
-        if positions is None:
-            raise RuntimeError('No positions returned from MT5. Error is ' + str(mt5.last_error() or ''))
-        return positions
+    def get_positions(self) -> dict: # Note on 1.21.24 that MT5 returns positions as a named Tuple
+        pos = mt5.positions_get()
```

Comment:
This code has lost the runtime error when mt5 returns an error. We'll want that to keep us safe from bugs and help us debug.

Reply:
Fixed